ccxt.js

```diff
@@ -1,234 +0,0 @@
-"use strict";
```

Comment:
This file is now under `js/ccxt.js` but git does not recognize the move... so we can remove it